tests/Functional/BasePantherTestCase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Tests\Functional;
+
+use DAMA\DoctrineTestBundle\Doctrine\DBAL\StaticDriver;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\SchemaTool;
+use Symfony\Component\Panther\PantherTestCase;
+
+abstract class BasePantherTestCase extends PantherTestCase
```

Comment:
We can reuse this logic for all panther tests (even if I recommend to add as less as possible tests that run with real browser given how much it affects to performance)

Only do panther tests if you need to interact with the page and it includes javascript, otherwise webtestcase are good enough

Reply:
For me there are a few cases for this bundle, maybe only the compose page and maybe the collection type of the container block